buildpublish.gradle

```diff
@@ -106,4 +105,13 @@ tasks.withType(FindBugs) {
         xml.enabled = false
         html.enabled = true
     }
-}
\ No newline at end of file
+}
+
+apply plugin: 'jacoco'
```

Comment:
Should probably be in a separate PR (adding build + test coverage). But not too worry. I'll roll all of these into a new branch anyway.

Reply:
I merged into my master... weird how it got put here